### PR TITLE
[SKIP CI] [WIP] Creating a new CI/CD pipeline using Jenkins

### PR DESCRIPTION
**Pull request by ashahi1**
This PR contains a Jenkinsfile for creating a new CI/CD pipeline using Jenkins.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
+node("vdvs-slave-two") {
+
+    def app
+    
+    /*def DIRECTORY = 'docker-volume-vsphere'*/
+
+    def ERROR = 'false'
+   
+    dir('docker-volume-vsphere') {
```
Comment (shuklanirdesh82) [inline Jenkinsfile]: let's get started using the new name and avoid referring `docker-volume-vsphere`

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+       sh "echo Deleting \\${JOB_NAME} "
+       deleteDir()
+    }
+   
+        stage('Clone repository') {
```
Comment (shuklanirdesh82) [inline Jenkinsfile]: better way to put is `checkout code` or `checkout PR`

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        /* Let's make sure we have the repository cloned to our workspace */
+            checkout scm
+        }
+
+        stage('Build Linux plugin') {
```
Comment (shuklanirdesh82) [inline Jenkinsfile]: how about using `build artifacts` or `build binaries`?

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,79 @@
+node("vdvs-slave-two") {
+
+    def app
+   
+    def ERROR = 'false'
+   
+    dir('docker-volume-vsphere') {
```
Comment (shuklanirdesh82) [inline Jenkinsfile]: let's get started using the new name and avoid referring `docker-volume-vsphere`

**Comment by ashahi1** [comment]
@shuklanirdesh82 Yes, will update it. I missed it.

**Comment by ashahi1** [comment]
retest this please

**Pull request merged**